src/pytezos/cli/cli.py

```diff
@@ -173,5 +185,76 @@ def deploy(
             logger.info(status)
 
 
+@cli.command(help='Run containerized sandbox')
+@click.option('--protocol', type=click.Choice(['Florence', 'Edo']), help='Protocol to use', default='Florence')
```

Comment:
I suggest to use multiple aliases here and pass value as is if none matched. @m-kus, could we call `edo2` protocol just `edo`?

Reply:
Yes, we can